src/tests/integ/mongo_interaction_test.go

```diff
@@ -0,0 +1,64 @@
+package mongo_interaction_test
+
+import (
+	"fenix/src/database"
+	"fenix/src/test_utils"
+	"fenix/src/test_utils/test_client"
+	"fenix/src/websocket_models"
+	"log"
+	"testing"
+
+	"github.com/joho/godotenv"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+func TestYodelIntegration(t *testing.T) {
+	if testing.Short() {
+		t.Skip()
+	}
+
+	t.Run("yodel creation results in new db entry", func(t *testing.T) {
+		env, err := godotenv.Read("../../../.env")
```

Comment:
Relative paths seem easily breakable. Is there some way we could figure out the absolute path to the root of the project directory and make it accessible across the project?